plugin/evm/service.md

```diff
@@ -1,9 +1,5 @@
----
-title: Subnet-EVM API
----
-
```

Comment:
Please revert this, this might be consumed by a documentation engine somewhere else